azimuth/config.py

```diff
@@ -19,6 +20,44 @@
 REQUIRED_EXT = {"csv", "json"}
 
 
+class SupportedLanguage(str, Enum):
+    en = "en"
+    fr = "fr"
+
+
+# spaCy models must be in pyproject.toml to be loaded by Azimuth
+class SupportedSpacyModels(str, Enum):
+    en_core_web_sm = "en_core_web_sm"
+    fr_core_news_md = "fr_core_news_md"
+
+
+class LanguageDefaultValues(AliasModel):
+    suffix_list: List[str]
+    prefix_list: List[str]
+    spacy_model: str
```

Comment:
So here we want a `SupportedSpacyModels`, right? Not any `str`. Does that work?
```suggestion
    spacy_model: SupportedSpacyModels
```

Reply:
Oh, yes, thanks for catching that!